workers/unity/Packages/com.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -99,16 +108,33 @@ public static void Launch()
                 else
                 {
                     var content = File.ReadAllText(latestLogFile.FullName);
-                    message = $"{message}\n{content}";
-
-                    Debug.LogError(message);
+                    Debug.LogError($"{message}\n{content}");
                 }
 
                 process.Dispose();
                 process = null;
             };
         }
 
+        private static string GetCommand()
+        {
+            if (Application.platform == RuntimePlatform.OSXEditor)
+            {
+                return "osascript";
+            }
+            return Common.SpatialBinary;
+        }
+
+        private static string GenerateCommandArgs(string command)
+        {
+            string generatedCommandArgs = command;
+            if (Application.platform == RuntimePlatform.OSXEditor)
+            {
+                generatedCommandArgs = $"-e 'tell application \"Terminal\"\nactivate\ndo script \"cd {SpatialProjectRootDir} && {Common.SpatialBinary} {command}\"\nend tell'";
```

Comment:
Driveby opportunity 😈 : while we're touching this can we convert this to a verbatim interpolated string please? 

```cs
                commandArgs = $@"-e 'tell application ""Terminal""
activate
do script ""cd {SpatialProjectRootDir} && {Common.SpatialBinary} local launch""
end tell'";
```

Reply:
Sure thing 🔫